modules/storage-persistent-storage-pv.adoc

```diff
@@ -179,8 +179,10 @@ endif::[]
 
 3. Use a recreate deployment strategy for pods that rely on AWS EBS.
 
+4. Only raw block volumes support ReadWriteMany access mode for Fibre Channel and iSCSI. For more information, see "Block volume support".
```

Comment:
```suggestion
4. Only raw block volumes support the ReadWriteMany (RWX) access mode for Fibre Channel and iSCSI. For more information, see "Block volume support".
```

Reply:
I think here we can asssign the abbreviation for "ReadWriteMany" and later we can continue using RWX.

Reply:
done